PROJECT_GUIDE.md

```diff
@@ -0,0 +1,37 @@
+# Goals/Mission
+
+The mission is to improve developer relations, growth in proficiency, and network for people within LagosJUG and beyond. Feel free to dive into our open-source ecosystem, where contributions are driven by passion and collaboration.
+
+# Project
+A Project at any point in time must have a Lead/Owner and at least 3 team members working to achieve a milestone within its defined scope under the supervision of the Lagos Admin. The Project Owner should document Each Milestone as a user story.
```

Comment:
Under the supervision of the Lagos JUG Admin. I think this statement should read 'under the supervision of the **Lagos JUG Admin, as the former could mean an Admin resident in Lagos.

Attention should be given to the use of capital letters. The statement is continuous, so "Each" should be replaced with "each, small letter 'e'"  The project owner should document each milestone as a user story.